samples/subsys/shell/shell_module/prj_minimal_rtt.conf

```diff
@@ -18,4 +18,7 @@ CONFIG_SHELL_HELP_ON_WRONG_ARGUMENT_COUNT=n
 CONFIG_SHELL_STATS=n
 CONFIG_SHELL_CMDS=n
 CONFIG_CONSOLE=y
-CONFIG_SHELL_BACKEND_RTT=y
+
+#enable RTT shell
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_SHELL_BACKEND_RTT=y
```

Comment:
Missing newline

Reply:
Carles there are plenty of PRs removing empty line at the end of file. I am confused to be honest.

Reply:
done